client/src/index.test.ts

```diff
@@ -112,7 +114,7 @@ Deno.test("remote cache timeout", async () => {
 Deno.test("local variadic cache", () =>
   testVariadic(localCache({ id: "some-id" })));
 
-Deno.test("memory variadic cache", () => testVariadic(memCache));
+Deno.test("memory variadic cache", () => testVariadic(memCache({})));
```

Comment:
Maybe make this an optional param?

Reply:
I have this agenda about optional and default args

I will explain on the phone